sci-electronics/geda/geda-1.9.2-r1.ebuild

```diff
@@ -65,36 +66,30 @@ src_prepare() {
 	sed -i -e 's/sarlacc_schem_LDFLAGS =/sarlacc_schem_LDFLAGS = $(GIO_LIBS)/' \
 		contrib/sarlacc_schem/Makefile.am || die
 
+	rm docs/wiki/media/geda/gsch2pcb-libs.tar.gz || die
+
 	eautoreconf
 }
 
 src_configure() {
-	econf \
-		$(use_enable threads threads posix) \
-		$(use_with stroke libstroke) \
-		$(use_enable nls) \
-		$(use_enable debug assert) \
-		--disable-doxygen \
-		--disable-rpath \
+	local myconf=(
+		--disable-doxygen
+		--disable-rpath
 		--disable-update-xdg-database
+		$(use_enable debug assert)
+		$(use_enable nls)
+		$(use_enable threads threads posix)
+		$(use_with stroke libstroke)
+	)
+
+	econf "${myconf[@]}"
```

Comment:
you can keep it like this now, but I generally consider this `myconf` thing a relic from `autotools-utils` days.

Reply:
Oki, it just seemed a bit more clean like this rather than having backslashes